Doppler.Currency/Startup.cs

```diff
@@ -33,8 +32,7 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllers()
-                .AddJsonOptions(options => { options.JsonSerializerOptions.IgnoreNullValues = true; });
+            services.AddControllers();
```

Comment:
Why did you remove `IgnoreNullValues` configuration? it is by default now? are you not changing the behavior?

Reply:
Because he is obsolete and not necessary in this API

Reply:
Is it the default behavior now?

Reply:
Yes

Reply:
jeje sorry without this line `options.JsonSerializerOptions.IgnoreNullValues = true;` the default behavior is `false`(change de behavior), I think that is not necessary for this API to ignore null values, but I can do a change in another pull request because that is not having a relationship with update net6